hacspec-scrambledb/scrambledb/Cargo.toml

```diff
@@ -5,9 +5,42 @@ edition = "2021"
 
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
+[lib]
+crate-type = ["rlib", "cdylib"]
+
 [dependencies]
 oprf.workspace = true
 elgamal.workspace = true
 prp.workspace = true
 p256.workspace = true
 hacspec_lib.workspace = true
+hash-to-curve.workspace = true
+
+wasm-bindgen = { version = "0.2.87", optional = true }
+rand = { version = "0.8.5", optional = true }
+getrandom = { version = "0.2.10", features = ["js"], optional = true }
+hex = { version = "0.4.3", optional = true }
+
+
+gloo-utils = { version = "0.1", features = ["serde"] }
+serde_json = "1.0.106"
+
+[dependencies.web-sys]
```

Comment:
Is there a reason to specify this separately here?

Reply:
I put it separately since I have to specify all these features for `web-sys`, so I thought this was easier to read.

Reply:
Formatting should take care of that. But this is fine too.